src/button.test.visuals.ts

```diff
@@ -91,6 +63,7 @@ test.describe('Button', () => {
     test.describe('variant', () => {
       test('variant="primary"', async ({ page }, test) => {
         await page.goto('/iframe.html?id=button--button');
+        await page.waitForSelector('glide-core-button');
```

Comment:
We always have to wait for components. Took me a second to figure out why only two tests were failing.